.travis.yml

```diff
@@ -9,8 +9,11 @@ go:
   - 1.5
 
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - echo "Dpkg::Options { \"--force-confdef\"; \"--force-confold\"; }" |sudo tee -a /etc/apt/apt.conf.d/local
```

Comment:
Call me neurotic, but I think there should be a space on both sides of a pipe (`|`).